### ISSUE TITLE
Python docstring formatting

### ISSUE THREAD

**Issue by theairbend3r** [danymat/neogen#71]
Hi, Neogen adds a string "binary_operator" in the `Returns` part of the docstring when there's no annotation. I think it shouldn't add anything if a return type annotation is not specified.

### Issue 1

```
def lol(x, y):
  """

  Parameters
  ----------
  x : 
   
  y : 
   

  Returns
  -------    
  binary_operator

  """ 
  return x + y 
```

It works properly when the function has an annotation for return type.

```
  def lol(x, y) -> str:    
      """    
      
      Parameters    
      ----------    
      x :     
              
      y :     
              
      
      Returns    
      -------    
      str    
              
      """    
      return x + y 
```

### Issue 2

I've also noticed that `*args, **kwargs` don't show up. 

The current behaviour is:
```
def lol(x, y, *args, **kwargs) -> str:
    """                                                      
  
    Parameters  
    ----------  
    x :   
          
    y :   
         
  
    Returns  
    -------  
    str  
          
    """  
    return x + y  
```

For correct behaviour, see [def module_level_function(param1, param2=None, *args, **kwargs):](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html) in the Sphinx's NumPy style Python docstrings. It is basically - 

```
def module_level_function(param1, param2=None, *args, **kwargs):
    """This is an example of a module level function.

    Function parameters should be documented in the ``Parameters`` section.
    The name of each parameter is required. The type and description of each
    parameter is optional, but should be included if not obvious.

    If ``*args`` or ``**kwargs`` are accepted,
    they should be listed as ``*args`` and ``**kwargs``.

    The format for a parameter is::

        name : type
            description

            The description may span multiple lines. Following lines
            should be indented to match the first line of the description.
            The ": type" is optional.

            Multiple paragraphs are supported in parameter
            descriptions.

    Parameters
    ----------
    param1 : int
        The first parameter.
    param2 : :obj:`str`, optional
        The second parameter.
    *args
        Variable length argument list.
    **kwargs
        Arbitrary keyword arguments.

    Returns
    -------
    bool
     
    """
    if param1 == param2:
        raise ValueError('param1 may not be equal to param2')
    return True

```

**Comment by danymat** [comment]
Hello !

> I think it shouldn't add anything if a return type annotation is not specified.

So you think that we should remove the automatic detection of a `return` statement, and only look for type annotation ? What about python versions `< python3` where type hints are not featured?
In the meantime, I fixed the issue 1 in 11c8ca4.

> I've also noticed that *args, **kwargs don't show up.
I added support for these in last commit: 3ac1e14

Can you try those changes ?

**Comment by theairbend3r** [comment]
> So you think that we should remove the automatic detection of a return statement, and only look for type annotation ? What about python versions < python3 where type hints are not featured?

I just meant that the string 'binary_operator' shouldn't be added if there's no type annotation. Sorry for the confusing wording. Automatic detection of `return` statement is actually a very useful feature! :smile: 

> Can you try those changes ?

I can confirm both issues are solved in the latest version!
